src/main/java/com/github/sgov/server/controller/VocabularyController.java

```diff
@@ -42,11 +42,11 @@ public VocabularyController(VocabularyRepositoryService vocabularyService) {
     @GetMapping(produces = {
         MediaType.APPLICATION_JSON_VALUE, JsonLd.MEDIA_TYPE})
     @ApiOperation(value = "Retrieve all vocabularies.")
-    public List<VocabularyDto> findAll(@RequestHeader
+    public List<VocabularyWorkspacesDto> findAll(@RequestHeader
```

Comment:
I would guess that this cannot work with `JsonLd.MEDIA_TYPE` as JSON-LD serializer would not be able to serialize WorkspaceDto and relation inWorkspaces as they are not annotated by Jopa.

If so I suggest to annotate it by JOPA. (see e.g. UserAccount entity for that)